### PR TITLE
Document the control algorithm from Confluence

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: doc/ca-in-ofc.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
+#############################################
+Control Algorithm in Optical Feedback Control
+#############################################
+
+The optical feedback control (OFC) estimates the offset (:math:`\vec{u}`) of degree of freedom (DOF) at time :math:`k+1` based on the wavefront error(:math:`\vec{y}_{k}`) at time :math:`k`.
```
Comment (teweitsai) [inline doc/ca-in-ofc.rst:5]: error (:math:`\vec{y}_{k}`)

File: doc/ca-in-ofc.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+The feedback amplitude is based on the gain(:math:`g`).
+The first step is to evaluate the optical state(:math:`\vec{x}_{k}`) at time :math:`k` in the basis of DOF.
```
Comment (teweitsai) [inline doc/ca-in-ofc.rst:7]: state (:math:`\vec{x}_{k}`)

File: doc/ca-in-ofc.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+.. math::
+
+    \begin{align*}
+    \vec{x}^{T}\textbf{Q}\vec{x} &= (\textbf{C}(\vec{y} - \vec{y}_{2} - \vec{y}_{c}(l)))^{T}\textbf{C}(\vec{y} - \vec{y}_{2} - \vec{y}_{c}(l)) \\ 
+    &= (\vec{y} - \vec{y}_{2} - \vec{y}_{c}(l))^{T}\textbf{C}^{T}\textbf{C}(\vec{y} - \vec{y}_{2} - \vec{y}_{c}(l)) \\
+    &= (\textbf{A}\vec{x})^{T}\textbf{C}^{T}\textbf{C}\textbf{A}\vec{x} \\
+    &= \vec{x}^{T}\textbf{A}^{T}\textbf{C}^{T}\textbf{C}\textbf{A}\vec{x}.
+    \end{align*}
+
+This means :math:`textbf{Q} = \textbf{A}^{T}\textbf{C}^{T}\textbf{C}\textbf{A}`.
```
Comment (teweitsai) [inline doc/ca-in-ofc.rst:95]: `textbf{Q}` failed at:
https://ts-ofc.lsst.io/v/PR-33/ca-in-ofc.html

File: doc/ca-in-ofc.rst [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+Consider three different reference points "x0, "0", and "x00". "x0" means the offset will only trace the previous one; "0" means the offset will trace the real value and target for 0; "x00" means the offset will only trace the relative changes of offset without regarding the real value.
+Assume the telescope's state is :math:`\vec{s}_{0}` in the time :math:`k=0` and :math:`\vec{s}` in time :math:`k`, and we can intentionally shift :math:`\vec{m}_{x}` to be:
+
+.. math::
+
+    \begin{align*}
+    \vec{m}_{x} &\rightarrow \vec{m}_{x}, \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \  \text{for x0} \\
+    &\rightarrow \vec{m}_{x} + \rho^{2}\textbf{H}\vec{s}, \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \text{for 0} \\
+    &\rightarrow \vec{m}_{x} + \rho^{2}\textbf{H}(\vec{s} - \vec{s}_{0}), \ \ \ \text{for x00}.
```
Comment (teweitsai) [inline doc/ca-in-ofc.rst:140]: We could try to align the  \text{for x0}, \text{for 0}, and \text{for x00} to make them look better on lsst.io.

**Review by teweitsai — APPROVED** [review]
The generated document looks good to me. It looks like the rst support the latex math well (better than the confluence page, which I know `&=` is not supported in my memory. Thanks!

**Review by tribeiro — CHANGES_REQUESTED** [review]
Can you please move these docs to the developer-guide space?

File: doc/ca-in-ofc.rst [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+Consider the correction of wavefront error between center and corcer, we rewrite the above equation as:
+
+.. math::
+
+    \begin{align*}
+    \vec{u} &= - \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \textbf{F} \sum_{i}w_{i} \textbf{M}_{i}^{T}\text{diag}(\vec\alpha)_{i}^{2} (\textbf{M}_{i} \vec{x}_{k, i} +\vec{y}_{2, i}) \\
+    &= - \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \textbf{F} \sum_{i}w_{i} \textbf{M}_{i}^{T}\text{diag}(\vec\alpha)_{i}^{2} \vec{m}_{x, i}.
+    \end{align*}
+
+Consider three different reference points "x0, "0", and "x00". "x0" means the offset will only trace the previous one; "0" means the offset will trace the real value and target for 0; "x00" means the offset will only trace the relative changes of offset without regarding the real value.
```
Comment (tribeiro) [inline doc/ca-in-ofc.rst:132]: Can you break this down into a "definitions list"?

```
Consider three different reference points ``x0``, ``0``, and ``x00``. 

  ``x0``
    The offset will only trace the previous one;
  ``0``
    The offset will trace the real value and target for ``0``
  ``x00``
    The offset will only trace the relative changes of offset without regarding the real value.
```

File: doc/ca-in-ofc.rst [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+Control Algorithm in Optical Feedback Control
+#############################################
+
+The optical feedback control (OFC) estimates the offset (:math:`\vec{u}`) of degree of freedom (DOF) at time :math:`k+1` based on the wavefront error (:math:`\vec{y}_{k}`) at time :math:`k`.
+The feedback amplitude is based on the gain (:math:`g`).
+The first step is to evaluate the optical state (:math:`\vec{x}_{k}`) at time :math:`k` in the basis of DOF.
+The second step is to evaluate :math:`\vec{u}`.
+The final step is to decide the feedback amplitude based on :math:`g`.
+
+The optical state is defined in the following:
```
Comment (tribeiro) [inline doc/ca-in-ofc.rst:11]: in the following ->  as

**Review by tribeiro — APPROVED** [review]
Looks great! Thanks for the updates!

**Pull request merged**